build-libs/generate-variant-rewrites.js

```diff
@@ -0,0 +1,92 @@
+const createFetch = require('@vercel/fetch')
+const fetch = createFetch()
+
+const MAX_LIMIT = '100' // Defaults to API max
+
+async function fetchAll({ after, fetchedTutorials }) {
```

Comment:
Copied some of this logic over from the learn client because I was wrestling with modules 😭 😆

Reply:
Putting this into a script that we call with `hc-tools` would prob make it possible to import from the learn client, fwiw.

Reply:
ohh thats right, i forgot about the `hc-tools` magic. okay i'll check that out.

Reply:
Done!